packages/core/src/App/Components/Layout/Header/account-info-wrapper.tsx

```diff
@@ -5,17 +5,23 @@ import { isBot } from '@deriv/shared';
 type TAccountInfoWrapper = {
     is_disabled?: boolean;
     is_mobile?: boolean;
+    is_dtrader_v2?: boolean;
     disabled_message?: string;
 };
 
 const AccountInfoWrapper = ({
     is_disabled,
     is_mobile,
+    is_dtrader_v2,
     disabled_message,
     children,
 }: React.PropsWithChildren<TAccountInfoWrapper>) =>
     is_disabled && disabled_message ? (
-        <Popover alignment={isBot() && is_mobile ? 'bottom' : 'left'} message={disabled_message} zIndex='99999'>
+        <Popover
+            alignment={(isBot() || is_dtrader_v2) && is_mobile ? 'bottom' : 'left'}
```

Comment:
On current production we have logic for disabling account switcher (all, not only for a particular account). In Figma design of DTrader-V2 this case is not handled. 

While design doesn't exist (temporary), I've added condition for old file in order to preserve logic of disabling for redesign too.